ecms_base/features/custom/ecms_hotel/config/install/media.type.hotel_image.yml

```diff
@@ -0,0 +1,12 @@
+langcode: en
+status: true
+dependencies: {  }
+id: hotel_image
+label: 'Hotel Image'
+description: 'Images for accommodations.'
+source: image
+queue_thumbnail_downloads: false
+new_revision: false
+source_configuration:
+  source_field: field_media_image_2
```

Comment:
This is the same source ID as in #28. Is there a potential for a conflict or dependency problem with this?

Reply:
I think you're right - I haven't had both content types enabled simultaneously. Going to perform some more testing locally, but it looks like #28 is ready to go, so once that gets merged in I will ensure both can be enabled.

Reply:
Ok was able to replace that with a hotel specific one